src/aframe-streetmix-parsers.js

```diff
@@ -693,30 +688,29 @@ function createCenteredStreetElement (segments) {
   return streetEl;
 }
 
-function createSegmentElement (scaleX, positionX, positionY, rotationY, mixinId, length, repeatCount, elevation = 0) {
+function createSegmentElement (scaleX, positionY, rotationY, mixinId, length, repeatCount, elevation = 0) {
   var segmentEl = document.createElement('a-entity');
   const scaleY = length / 150;
 
   const scalePlane = scaleX + ' ' + scaleY + ' 1';
   const scaleBox = scaleX + ' 1 1';
+  let heightLevels = [0.2, 0.4, 0.8]
+  let height = heightLevels[elevation];
+  if (elevation === 0) {
+    positionY = -0.1;
```

Comment:
This discards some positionY that may have been set earlier in processSegments such as for separator segments that set positionY = 0.01 (to prevent z fighting from 2 nearby planes) @Algorush